communicator/mesoshttp/mesosapi.go

```diff
@@ -0,0 +1,73 @@
+package mesoshttp
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/vmturbo/mesosturbo/communicator/metadata"
+	"github.com/vmturbo/mesosturbo/communicator/util"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type MesosHTTPClient struct {
+	MesosMasterBase string
+}
+
+func (mesos *MesosHTTPClient) MesosPostRequest(metadata *metadata.ConnectionClient, dcos_token string) error {
+
+	var jsonStr []byte
+	url := "http://" + metadata.MesosIP + "/acs/api/v1/auth/login"
+
+	if dcos_token == "" {
+		fmt.Println(`{"uid":"` + metadata.DCOS_Username + `","password":"` + metadata.DCOS_Password + `"}`)
+		jsonStr = []byte(`{"uid":"` + metadata.DCOS_Username + `","password":"` + metadata.DCOS_Password + `"}`)
+	} else {
+		fmt.Println(`{"uid":"` + metadata.DCOS_Username + `","password":"` + metadata.DCOS_Password + `","token":"` + dcos_token + `"}`)
+		jsonStr = []byte(`{"uid":"` + metadata.DCOS_Username + `","password":"` + metadata.DCOS_Password + `","token":"` + dcos_token + `"}`)
+	}
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer(jsonStr))
+	req.Header.Set("Content-Type", "application/json")
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+
+	defer resp.Body.Close()
+	if err != nil {
+		fmt.Printf("Error in POST request: %s", err)
+		return err
+	} else {
+		// Get token if response if OK
+		if resp.Status == "" {
+			fmt.Println("Empty response status")
+			return err
+		}
+		fmt.Printf(" Status is : %s \n", resp.Status)
+		statusmsg := strings.Split(resp.Status, " ")
+		if statusmsg[0] == "200" {
+			fmt.Println(" We got 200, response Status:", statusmsg[0])
```

Comment:
Delete this comment, it is not very helpful if everything is working as expected